### PR TITLE
report is_on state and call the correct device

### PR DESCRIPTION
**Pull request by ntcong**
entity should report is_on state so HA can call on/off correctly

also _device should be _climate

### CODE REVIEW

File: custom_components/climate/xiaomi_miio.py [hunk 1]
```diff
@@ -368,6 +370,11 @@ def fan_list(self):
         """Return the list of available fan modes."""
         return self._fan_list
 
+    @property
+    def is_on(self) -> bool:
+        """Return True if the entity is on"""
+        return self._is_on
```
Comment (syssi) [inline custom_components/climate/xiaomi_miio.py]: Please return self._state here.

File: custom_components/climate/xiaomi_miio.py [hunk 2]
```diff
@@ -228,6 +229,7 @@ def async_update(self):
             _LOGGER.debug("Got new state: %s", state)
 
             self._state = state.is_on
+            self._is_on = state.is_on
```
Comment (syssi) [inline custom_components/climate/xiaomi_miio.py]: This isn't needed anymore if you use `self._state`.

File: custom_components/climate/xiaomi_miio.py [hunk 3]
```diff
@@ -132,6 +132,7 @@ def __init__(self, hass, name, host, token, sensor_entity_id, customize):
         self._current_swing_mode = None
         self._current_operation = None
         self._current_fan_mode = None
+        self._is_on = False
```
Comment (syssi) [inline custom_components/climate/xiaomi_miio.py]: This isn't needed anymore if you use `self._state`. The initial state must be `None` to mark the device as unavailable. The first update call will initialize the state.

**Comment by ntcong** [comment]
I have updated the commit, I thought that _state can be changed outside so I didn't use it.

**Pull request merged**